cw_decred/.metadata

```diff
@@ -1,10 +1,36 @@
 # This file tracks properties of this Flutter project.
 # Used by Flutter tool to assess capabilities and perform upgrades etc.
 #
-# This file should be version controlled and should not be manually edited.
+# This file should be version controlled.
```

Comment:
Did you change these manually?

Also what is the commit below? f468f3366c26a5092eb964a230ce7892fda8f2f8